web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js

```diff
@@ -132,33 +132,24 @@ export const MachineLearningSettings = () => {
           backends={backends}
         />
 
-        <Divider height={32} />
-
         {backends.length > 0 && (
-          <Description style={{ marginTop: 0, maxWidth: 680, lineHeight: "1.5em" }}>
-            A connected model has been detected! If you wish to fetch
-            predictions from this model, please follow these steps:
-            <br />
-            <br />
-            1. Navigate to the <i>Data Manager</i>.<br />
-            2. Select the desired tasks.<br />
-            3. Click on <i>Batch predictions</i> from the{' '}
-            <i>Actions</i> menu.
-            <br />
-            <br />
-            Additionally, you can configure the system to use this model for
-            fetching live predictions in {" "}
-            <a
-              style={{ color: "unset", textDecoration: "underline" }}
-              target="_blank" href="https://labelstud.io/guide/labeling.html#Choose-which-tasks-to-label"
-            ><i>Label&nbsp;All&nbsp;Tasks</i></a>{" "}
-            and{" "}
-            <a
-              style={{ color: "unset", textDecoration: "underline" }}
-              target="_blank" href="https://labelstud.io/guide/get_started#Interface"
-            ><i>Quick View</i></a>{" "}
-            on the <i>Annotation</i> tab.
-          </Description>
+          <>
+            <Description>
+              A connected model has been detected! If you wish to fetch
+              predictions from this model, please follow these steps:
+              <br />
+              <br />
+              1. Navigate to the <i>Data Manager</i>.<br />
+              2. Select the desired tasks.<br />
+              3. Click on <i>Retrieve predictions</i> from the <i>Actions</i> menu.
```

Comment:
I see this is copy/paste from before but why are we relying on `<br />` for spacing and layout? also this seems like an optimal spot to use `<ol>` for the list - can we get a follow up ticket to address this?

Reply:
I'm going to fix this asap, yes, thank you!